README.md

```diff
@@ -75,8 +75,8 @@ Some people may decide the attack scenario above is worth the usability tradeoff
 You must have Xcode Command Line Tools installed to build this project.
 
 ```bash
-# Install Commaned Line Tools
-xcode-select --install
```

Comment:
The capitalizating here is correct.

Reply:
Usually you don't capitalize for shell commands